stable/traefik/values.yaml

```diff
@@ -310,6 +311,7 @@ dashboard:
   service: {}
     # annotations:
     #   key: value
+  ingressEnabled: true
```

Comment:
This should also be optional (commented out by default)

Reply:
The way chart is done - I think it is difficult to achieve without bigger changes, I thought to do less disruption as possible. Unless false becomes default value of cause.